cmake/configure/configure_1_threads.cmake

```diff
@@ -138,6 +138,24 @@ MACRO(FEATURE_THREADS_FIND_EXTERNAL var)
   IF(TBB_FOUND)
     SET(${var} TRUE)
   ENDIF()
+
+  #
+  # TBB versions before 4.2 are missing some explicit calls to std::atomic::load
+  # in ternary expressions; these cause compilation errors in some compilers
+  # (such as GCC 8.1 and newer). To fix this we simply blacklist all older
+  # versions:
+  #
+  IF(TBB_VERSION VERSION_LESS "4.2")
+    MESSAGE(STATUS "The externally provided TBB library is older than version\n"
+      "4.2.0, which is the oldest version compatible with deal.II and its\n"
+      "supported compilers."
```

Comment:
@bangerth The string should be printed on a single line, so @drwells please remove the manual line breaks above.

Reply:
If you do that, please also consider indenting by two spaces to make it clear that the text is subservient to the previous line.